src/picongpu/include/particles/particleToGrid/derivedAttributes/ChargeDensity.def

```diff
@@ -35,6 +36,22 @@ namespace derivedAttributes
         HDINLINE float1_64
         getUnit() const;
 
+        HDINLINE std::vector<float_64>
+        getUnitDimension() const
+        {
+           /* L, M, T, I, theta, N, J
+            *
+            * ChargeDensity is in Coulomb / cubic meter: Q / m^3 = I * t / m^3
+            *   -> L^-3 * T * I
+            */
+           std::vector<float_64> unitDimension( 7, 0.0 );
+           unitDimension.at(0) = -3.0; // L^-3
```

Comment:
;) good quesiton and actually the same for `FieldJ`.

Currently, as in `getUnit()`, we just implement the 3D unit.